_data/file_formats/itm_v1/extended_header.yml

```diff
@@ -1,52 +1,118 @@
+    - 4 ⟶ <b><a name="itmv1_Extended_Header_0x0_4">Launcher</a></b>
+
+    For <code><a href="#itmv1_Extended_Header_0x2_3">Item</a></code> abilities:
+    - Do NOT use `0|None`, will crash the game.
+    - Only <code><a href="#itmv1_Extended_Header_0x0_3">Magical</a></code> abilities can be drained (along with <a href="../../opcodes/bgee.htm#op243">opcode #243</a>'s other restrictions).
+    - Type <code><a href="#itmv1_Extended_Header_0x0_4">4</a></code> cannot be used except from scripts (icon won't show up in actionbar).
+
+    For <code><a href="#itmv1_Extended_Header_0x2_1">Weapon</a></code> abilities:
+    - Do NOT use `0|None`, will crash the game.
+    - <code><a href="#itmv1_Extended_Header_0x0_1">Melee</a></code>: Ignores the <code><a href="#itmv1_Extended_Header_0x2A">projectile</a></code> field (always uses `#1|None`). Valid for Weapon Styles.
+    - <code><a href="#itmv1_Extended_Header_0x0_2">Ranged</a></code>:
+      - Utilizes <code><a href="#itmv1_Extended_Header_0x2A">projectile</a></code> field.
+      - Item cannot be equipped in <code><a href="../../files/ids/bgee/slots.htm#9">off-hand</a></code>, unless its <code><a href="#Header_ItemType">itemtype</a></code> can only be equipped in the <code><a href="../../files/ids/bgee/slots.htm#9">off-hand</a></code> slot, from <code><a href="../../files/2da/2da_bgee/itemtype.htm">ITEMTYPE.2DA</a></code>.
+      - If equipped in <code><a href="../../files/ids/bgee/slots.htm#9">off-hand</a></code>, it will attack as normal (at least for the offhand).
+      - <code><a href="../../files/ids/bgee/slots.htm#9">Off-hand</a></code> weapon will be suppressed while this item ability is selected in <code><a href="../../files/ids/bgee/slots.htm#35">mainhand</a></code> (EE only).
+    - <code><a href="#itmv1_Extended_Header_0x0_3">Magical</a></code>: Identical to `Type 1 (Melee)`, except that it can be drained.
+    - <code><a href="#itmv1_Extended_Header_0x0_4">Launcher</a></code>:
+      - In the <code><a href="../../files/ids/bgee/slots.htm#35">mainhand</a></code> slots:
+        - Ability cannot actually be selected, nor is the item considered "equipped" by script triggers or <a href="../../opcodes/bgee.htm#op183">opcode #183</a>.
+        - Equipped effects, stats and on-hit effects are added to the currently selected weapon ability IF that ability requires a <code><a href="#itmv1_Extended_Header_0x10">launcher</a></code> of this type.
+      - In the <code><a href="../../files/ids/bgee/slots.htm#9">off-hand</a></code> slot:
+        - Item cannot be equipped in <code><a href="../../files/ids/bgee/slots.htm#9">off-hand</a></code>, unless its <code><a href="#Header_ItemType">itemtype</a></code> can only be equipped in the <code><a href="../../files/ids/bgee/slots.htm#9">off-hand</a></code> slot, from <code><a href="../../files/2da/2da_bgee/itemtype.htm">ITEMTYPE.2DA</a></code>.
+        - Attack cycle will stop when this weapon would have attacked (must be re-initiated manually or by script).
```

Comment:
What does this last part mean? What cycle? Is it just the rndbase turn timing stuff?

Reply:
> What does this last part mean? What cycle? Is it just the rndbase turn timing stuff?

Clarified.